src/containers/EditSubjectFrontpage/components/SubjectFrontpageArticles.tsx

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright (c) 2020-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree. *
+ */
+
+import React, { FC, useEffect, useState } from 'react';
+import { injectT } from '@ndla/i18n';
+import { FieldHeader } from '@ndla/forms';
+import ElementList from '../../NdlaFilm/components/ElementList';
+import DropdownSearch from '../../NdlaFilm/components/DropdownSearch';
+import { ArticleType, TranslateType } from '../../../interfaces';
+import { getArticle } from '../../../modules/article/articleApi';
+import { fetchNewArticleId } from '../../../modules/draft/draftApi';
+
+interface Props {
+  t: TranslateType;
+  articles: string[];
+}
+
+const SubjectFrontpageArticles: FC<Props> = ({ t, articles }) => {
+  const [content, setContent] = useState<ArticleType[]>([]);
+
+  useEffect(() => {
+    const fetchSubject = async () => {
+      const externalIds = articles.map((x: any) => x.split(':').pop());
+      var articleIds: any[] = [];
+      var articleList: any[] = [];
+
+      await Promise.all(
+        externalIds.map(async externalId => {
+          try {
+            const id = await fetchNewArticleId(externalId);
+            articleIds.push(id.id);
+          } catch (e) {}
+        }),
+      );
```

Comment:
```suggestion
      const articleIds: string[] = await Promise.all(
        externalIds.map(async externalId => {
          try {
            return fetchNewArticleId(externalId);
          } catch (e) {}
        }),
      );
```
Går det an å gjøre noe sånt?

Reply:
den får det problemet med at jeg tydeligvis må ha try/catch, men enig i at det hadde vært mye bedre å ha noe mer sånt, så kan se på det!